synctl/cli.py

```diff
@@ -502,6 +512,92 @@ def pop_size_estimate(self):
         except ValueError as e:
             print(f"Exception: {e}")
 
+    def test_exec_estimate(self, tests, frequency, loc):
+        #The total test executions per month(30 days)
+       return tests * ((30 * 24 * 60) / frequency) * loc
```

Comment:
indention error, not four spaces
```suggestion
        return tests * ((30 * 24 * 60) / frequency) * loc
```